cmd/manager/aggregator.go

```diff
@@ -301,13 +301,13 @@ type compResultIface interface {
 
 func createOrUpdateOneResult(crClient aggregatorCrClient, owner metav1.Object, labels map[string]string, annotations map[string]string, exists bool, res compResultIface) error {
 	kind := res.GetObjectKind()
-
```

Comment:
nit: unrelated whitespace change?

Reply:
this might be from the autoformatting from my IDE